lib/async_hooks.js

```diff
@@ -151,15 +152,14 @@ class AsyncResource {
     this[async_id_symbol] = asyncId;
     this[trigger_async_id_symbol] = triggerAsyncId;
 
-    // This prop name (destroyed) has to be synchronized with C++
-    const destroyed = { destroyed: false };
-    this[destroyedSymbol] = destroyed;
```

Comment:
Is moving this safe?

Reply:
Yeah. This is only needed for the case where `requireManualDestroy = false`. We pass it on directly to C++ in `registerDestroyHook`.